src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -836,59 +832,6 @@ export default {
 			this.localValue = value
 		},
 
-		/**
-		 * Because FF have a decade old bug preventing contenteditable=false
-		 * to properly be deleted on backspace, we have to hack 👀
-		 * https://stackoverflow.com/a/59383394/3885878
-		 * https://stackoverflow.com/a/30574622
-		 *
-		 * @param {Event} event the delete keydown event
-		 */
-		onDelete(event) {
-			if (!this.isFF || !window.getSelection) {
-				return
-			}
```

Comment:
If you remove this method, `isFF` is also not needed anymore.

Reply:
Unrelated to pasting issue, but noticed:
if input is started or ended with embedded HTML (mention) `"@Test01 anyword @Test01"`:
- 🐞 in FF cursor isn't moving to the very start or end with ⬅️ ➡️ keys (⬆️ ⬇️ are working)
- 🐞 in Safari cursor jumps to the very beginning
- ✅ in Chrome everything is fine